tests/app/dao/test_organisation_dao.py

```diff
@@ -119,17 +119,53 @@ def test_update_organisation_domains_lowercases(
     assert {domain.domain for domain in organisation.domains} == expected_domains
 
 
-def test_add_service_to_organisation(notify_db, notify_db_session, sample_service, sample_organisation):
+def test_update_organisation_does_not_update_the_service_org_type_if_org_type_is_not_provided(
+    sample_service,
+    sample_organisation,
+):
+    sample_service.organisation_type = 'local'
```

Comment:
for pytest 4 compatibility, it's better to use `create_service()` and `create_organisation()` if you want to update them in the test, because updating fixtures in the test is deprecated in pytest 4 as far as I am aware.

Reply:
Changing the properties of a fixture is fine in Pytest 4 as long as the fixture is a parameter of the test function. It's calling fixtures like a function that you're no longer allowed to do. (I've tried out my changes in Pytest 4 to double check and there are no warnings)

Reply:
ahhh, ok, thanks for educating me 💖